src/openrct2/rct2/Limits.h

```diff
@@ -13,15 +13,15 @@
 namespace RCT2::Limits
 {
     using namespace RCT12::Limits;
-    constexpr uint8_t MaxStaff = 200;
-    constexpr uint8_t MaxBanners = 250;
+    constexpr uint8_t kMaxStaff = 200;
+    constexpr uint8_t kMaxBanners = 250;
     constexpr uint8_t kMaxTrainsPerRide = 32;
     constexpr uint8_t kMaxVehicleColours = 32;
     constexpr uint8_t kDowntimeHistorySize = 8;
-    constexpr uint16_t MaxEntities = 10000;
+    constexpr uint16_t kMaxEntities = 10000;
     constexpr uint16_t MaxEntitiesRCTCExtended = 15000; // Used in files marked with “classic flag” 0xF
```

Comment:
Might as well do all constants in this block.

Reply:
Why did you mark this as resolved? You haven’t addressed it yet.

Reply:
Sorry, I thought block meant the top constants clumped and not all of the constants. I have added the ones that I have missed.